src/bin/libint/CMakeLists.txt

```diff
@@ -28,6 +28,11 @@ else()
   set_target_properties(libint_compiler PROPERTIES COMPILE_FLAGS "-std=c99")
 endif()
 
+if(MSVC)
+    # Increase stack size from 1 MB to 4 MB
+    set_target_properties(libint_compiler PROPERTIES LINK_FLAGS "/STACK:4194304")
```

Comment:
If you've already had to increase the stack in test builds, please make sure that it can handle at least MAX_AM_ERI=6 and preferably 8.

Reply:
OK! I'll try.